src/style.css

```diff
@@ -0,0 +1,47 @@
+body {
+  margin: 0;
+}
+
+.moyo_header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.moyo_nav {
+  display: flex;
+}
+
+.link_nav {
+  position: relative;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 60px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: black;
+  margin-left: 20px;
+}
+
+.is-active {
```

Comment:
if we add this class to another element of the list?

![image](https://user-images.githubusercontent.com/44294945/64179538-02c89380-ce6c-11e9-98c7-3d58ebc63fce.png)